ulib/fs/VS/Makefile

```diff
@@ -11,13 +11,11 @@ ifeq ($(OS),Windows_NT)
 else
   FSC     = fsharpc
   # If can't find msbuild, use xbuild, but throw a warning
-  MSBUILD = $(shell which msbuild || (echo '\n\n\033[0;31mWarning:\033[0m could not find "msbuild", trying (deprecated) "xbuild"\n\n'>&2; which xbuild))
+  MSBUILD = $(shell which msbuild || (echo "dotnet msbuild"))
```

Comment:
Why not just `dotnet build`?

Reply:
I agree. The only reason why I do not replace, is that I do not know about "all" scenarios which involve "modern Mono". I try to survey people in the wild and seems to be it's safe choice. If @aseemr is support that - I can simplify build even further, because we would not need `msbuild.bat` altogether which is a win! and whole make file is just `dotnet build`

Reply:
That would be the best scenario. If you do that, I can try testing it on both windows and linux over the weekend.
We would also need someone from F* team to try it out on CI setup as I don't have access there.